src/vs/workbench/contrib/chat/browser/agentSessions/agentSessionsActions.ts

```diff
@@ -304,6 +307,34 @@ export class MarkAgentSessionReadAction extends BaseAgentSessionAction {
 
 //#endregion
 
+//#region Chat View Title Actions
+
+export class PickAgentSessionAction extends ViewAction<ChatViewPane> {
```

Comment:
I would prefer this to be a `Action2` and not `ViewAction` because its actually not running for the view, but another menu within.